apps/summerfi-api/lib/sdk-router-function/src/handlers/getRefinanceSimulation.ts

```diff
@@ -1,13 +1,14 @@
 import { z } from 'zod'
-import type { RefinanceParameters } from '@summerfi/sdk-common/orders'
 import { providerProcedure } from '~src/procedures/providerProcedure'
 import type { Position } from '@summerfi/sdk-common/common'
-import type { IPool } from '@summerfi/sdk-common/protocols'
+import type { LendingPool } from '@summerfi/sdk-common/protocols'
 import type { Simulation, SimulationType } from '@summerfi/sdk-common/simulation'
+import { refinace, type RefinanceDependencies } from '@summerfi/simulator-service'
+import type { RefinanceParameters } from '@summerfi/sdk-common/orders'
 
 const inputSchema = z.object({
   position: z.custom<Position>((position) => position !== undefined),
-  pool: z.custom<IPool>((pool) => pool !== undefined),
+  pool: z.custom<LendingPool>((pool) => pool !== undefined),
```

Comment:
I'm not sure if we should tie refinance to LendingPool here. In principle we could refinance any pool. The restriction should be imposed in the simulator and not here, in my opinion

Reply:
So again should I fix that in the Simulator, or @paszkowskiDamian should work on that

Reply:
If it was changed in Simulator I guess we need to discuss it with @paszkowskiDamian. Let's take it offline, I'll mark this as resolved